sashimi/gui/main_gui.py

```diff
@@ -113,7 +112,7 @@ def closeEvent(self, a0) -> None:
         self.wid_settings_tree.conf_window.close()
         self.wid_settings_tree.guide_window.close()
         self.st.wrap_up()
-        a0.accept()
+        # a0.accept()
```

Comment:
What is `a0` and why is it being passed to this method? What does `a0.accept()` do and why is it commented here?

Reply:
it should be like an "MouseEvent" obj or something like that. Commenting out should not do anything to guide behavior, but if keeping it I would need to pass a fake object with an accept() method.